script/release-flynn

```diff
@@ -116,15 +116,16 @@ main() {
     manifest="$(new_release_manifest)"
   fi
 
-  info "checking for existing release"
-  if jq --exit-status ".versions[] | select(.commit==\"${commit}\")" <<< "${manifest}"; then
-    fail "commit ${commit} has already been released"
-  fi
+  info "getting previous release tag"
+  local previous=$(git tag | tail -n 1)
```

Comment:
Hmm thinking about it, the ordering will break if we have more than 10 releases in a day (which could happen if say we have some intermittent issue that is blocking releases).

We can either zero pad the iteration (`printf "%02d" "${iteration}"`) or just count matching tags (`git tag --list "v${date}*" | wc -l`). I prefer the latter.